s4_design.toml

```diff
@@ -20,6 +20,24 @@ CMB_tensor_to_scalar_ratio = 0.005
 
 total_experiment_length_years = 7
 
+# Sensitivity factors impact only noise but not atmosphere
+# Copy-pasted from https://docs.google.com/spreadsheets/d/1jR9gSsJ0w1dEO5Jb_URlD3SWYtgFtwBgB3W88p6puo0/edit#gid=1776795091
+# on 13 April 2021
+# yield is given by detectors lost during fabrication
+# For pole SAT it includes yield and loss to uneven noise across detectors
+sensitivity_factor.Pole.SAT.LFS1  = 0.64 # 30 GHz
+sensitivity_factor.Pole.SAT.LFS2  = 0.64 # 40 GHz
+sensitivity_factor.Pole.SAT.MFLS1 = 0.64 # 85 GHz
+sensitivity_factor.Pole.SAT.MFLS2 = 0.52 # 145 GHz
+sensitivity_factor.Pole.SAT.MFHS1 = 0.64 # 95 GHz
+sensitivity_factor.Pole.SAT.MFHS2 = 0.52 # 155 GHz
+sensitivity_factor.Pole.SAT.HFS1  = 0.52 # 220 GHz
+sensitivity_factor.Pole.SAT.HFS2  = 0.52 # 270 GHz
+
+# Just consider yield
+sensitivity_factor.Chile.LAT.default = 0.8
+sensitivity_factor.Chile.SAT.default = 0.8
+sensitivity_factor.Pole.LAT.default = 0.8
```

Comment:
ok, sensitivity factors are now separated from the observing efficiencies, but have the same format.